src/main/xar-resources/data/docs/fn/xml-to-json.md

```diff
@@ -0,0 +1,41 @@
+# fn:xml-to-json() ([specification](https://www.w3.org/TR/xpath-functions-31/#func-xml-to-json"))
+handles most valid input according to specification. Notable exceptions follow.
+
+All (un-)escaping is delegated to com.fasterxml.jackson. All output is generated by jackson
+and should be proper JSON.
+
+`fn:xml-to-json()` does not replace codepoints in the range `1-31` or `127-159` in any string or map key.
+
+`fn:xml-to-json()` unescapes and reescapes any string and map key marked as being escaped.
+It does not do additional special character replacements mentioned in the spec.
+It does not do an otherwise verbatim copy.
+
+# Examples
```

Comment:
Yes this is much better. The examples actually explain the known limitations in way that is easy to understand. This is why code listings are often better then prose explanations.